src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir-book;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: lighter;
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: bold;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc) format("truetype");
+  font-weight: 300;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
```

Comment:
You can add `font-weight: 300;` to use by default for the whole document and `font-weight: inherit` for the `input` elements